graph-db/src/utils/getSecureParam.js

```diff
@@ -0,0 +1,22 @@
+const AWS = require('aws-sdk')
+
+let ssm
+
+/**
+ * getParam: Given token name, retrieve it from Parameter Store
+ * @param {String} param name of parameter to fetch
+ * @returns {JSON} server response object from Parameter Store
+ */
+exports.getSecureParam = async (param) => {
+  if (ssm == null) {
```

Comment:
Is `ssm` a singleton? Also should switch to `if (!ssm)` Should switch to singleton pattern if we're keeping track of a singular `ssm`

Reply:
Yes it is